client/src/pages/Tree/Tree.js

```diff
@@ -53,24 +57,26 @@ export default function Tree({
           <>
             <TreeHeader
               currentTreeData={currentTreeData}
-              vacant={vacant}
+              hasMissingData={hasUnfitData}
```

Comment:
This prop is misnamed.  The `TreeHeader` is looking for `hasUnfitData`, so the `!hasUnfitData` check there returns true and the like/adopt/edit buttons are shown even when the `hasUnfitData()` call above returned true.